Configuration/TypoScript/Library/themes.socialmedia.constantsts

```diff
@@ -15,69 +15,68 @@
 
 # Facebook
 #############################################
-# cat=socialmedia,advanced/socialmedia_facebook/; type=options[0,1]; label= Facebook: Activate?!
-themes.configuration.socialmedia.useFacebook = 1
+# cat=socialmedia,advanced/socialmedia_facebook/; type=options[Un-hide=1,Hide=0]; label= Facebook: Activate?!
+themes.configuration.socialmedia.useFacebook = Un-Hide
 
 # cat=socialmedia,advanced/socialmedia_facebook/; type=string; label= Facebook: Pagelink
 themes.configuration.socialmedia.facebookLink = https://www.facebook.com/typo3
 
 
 # Twitter
-# cat=socialmedia,advanced/socialmedia_twitter/; type=options[0,1]; label= Twitter: Activate?!
-themes.configuration.socialmedia.useTwitter = 1
+# cat=socialmedia,advanced/socialmedia_twitter/; type=options[Un-hide=1,Hide=0]; label= Twitter: Activate?!
```

Comment:
@danetkrueng Please replace Un-hide with Show.

Show/Hide is more understandable. And do the same for all occurrences in the file.